test/jasmine/tests/axes_test.js

```diff
@@ -3260,170 +3244,224 @@ describe('Test axes', function() {
         afterEach(destroyGraphDiv);
 
         it('should grow and shrink margins', function(done) {
+            var initialSize;
+            var previousSize;
+
+            function assertSize(msg, actual, exp) {
+                for(var k in exp) {
+                    var parts = exp[k].split(' -');
```

Comment:
Could you please elaborate why `' -'` (i.e. with leading space character) is the the separator used here?

Reply:
I thought it made the `check` calls very readable, for example:

```js
            .then(check('tilt y labels up 30 degrees', {'yaxis.tickangle': 30}, {
                t: 'grew', l: 'shrunk',
                b: '=', r: 'shrunk - as x ticks got shifted left'
            }))
```

Reply:
Nice. Could we use `|` there and apply split without a space character?

Reply:
Ok, I'm not sure how this makes things better, but I don't like arguing over stuff like that, so here's https://github.com/plotly/plotly.js/pull/3605/commits/18f827d738fb7ac48bee45f4b7f8f164ccec28a4

Reply:
Yes I see. Semicolon may be a better option there.